### PR TITLE
Compatibility with WordPress 5.8

### PR DESCRIPTION
**Pull request by hemant7405**
- Improvement: Updated the type of input field for authentication options.
- Improvement: Compatibility with WordPress 5.8

### CODE REVIEW

File: includes/class-affiliate-wp-track-external-visits.php [hunk 1]
```diff
@@ -432,7 +432,7 @@ public function callback_input_text( $args ) {
 
 		$value = isset( $options[ $args['name'] ] ) ? $options[ $args['name'] ] : '';
 		?>
-		<input style="width: 30%" type="text" id="<?php echo $args['id']; ?>" name="<?php echo CDTAWP_SETTINGS_GROUP; ?>[<?php echo $args['name']; ?>]" value="<?php echo $value; ?>"/>
+		<input style="width: 30%" type="password" id="<?php echo $args['id']; ?>" name="<?php echo CDTAWP_SETTINGS_GROUP; ?>[<?php echo $args['name']; ?>]" value="<?php echo $value; ?>"/>
```
Comment (patilvikasj) [inline includes/class-affiliate-wp-track-external-visits.php:435]: Variables missing escaping.

Comment (hemant7405) [reply includes/class-affiliate-wp-track-external-visits.php]: done

File: readme.txt [hunk 2]
```diff
@@ -46,7 +46,11 @@ No, you just need to select the plugin type as parent/child inside the plugin se
 
 == Changelog ==
 
-= Version 1.0.3 - Monday, 24th Aug 2020 =
+= Version 1.0.4 - Wednesday, 14th July 2021 =
```
Comment (Nikschavan) [inline readme.txt:49]: Why do we have changelog in changelog.txt as well as readme? which one does get displayed on W.org?

Comment (hemant7405) [reply readme.txt:49]: The readme file will be displayed on W.org

Comment (Nikschavan) [reply readme.txt:49]: Looks like the one from readme.txt is being used by WP.org - https://wordpress.org/plugins/cross-domain-tracker-for-affiliatewp/#developers

Can you confirm this, Let's only keep this in only one place.

Comment (hemant7405) [reply readme.txt:49]: Removed the changelog.txt file

**Pull request merged**